### DISCUSSION TITLE
can not seem to add category in storefront

### DISCUSSION THREAD

**Discussion by ayon120** [saleor/saleor#8279] [Q&A]
Hello, 
I am new to saleor and trying things out a bit. My saleor backend is on the heroku and running dashboard and storefront locally. 
I have created a new category named 'Homedelivery' from dashboard and added a product there. but when I run the storefront I can't seem to see that category. I saw that there is a navigation settings in dashboard under navbar menuitems where all the categories and subcategories are structures. But there when I tried to add, I did not seem to find the category I just created. But it is being shown in the categories part and also I can search it from graphql playground. If anyone can state what I am doing wrong that would be very helpful.
![Capture6](https://user-images.githubusercontent.com/30633846/135955464-350ce524-cbce-4322-9966-35086b5741be.JPG)
![Capture5](https://user-images.githubusercontent.com/30633846/135955491-9a81dc9e-3703-4768-954b-db72554713a0.JPG)

**Comment by karolkielecki** [comment]
@ayon120 Hi! Please start typing to begin search--you should be able to see your previously created category. I've just created `newcategory`, after typing I was able to select it. 

![image](https://user-images.githubusercontent.com/57907951/136926152-08978775-9b04-4cc8-a8bb-ac76e0b1e9d5.png)

**Reply by ayon120** [reply]
Thank you @karolkielecki for the solution. My system was very slow while running the storefront, hence I could not find the solution then but now it is alright. Can you please take a look at my recent question that I uploaded a few minutes back? I will be very grateful.
